c10/Device.h

```diff
@@ -110,3 +110,8 @@ struct hash<at::Device> {
   }
 };
 } // namespace std
+
+// TODO: Remove when we add global namespace include
+namespace at {
+using c10::Device;
```

Comment:
I don't quite get it why, but windows build failed with this:

19:01:37 C:\Jenkins\workspace\pytorch-builds\pytorch-win-ws2016-cuda9-cudnn7-py3-build\c10/Device.h(116): error C2874: using-declaration causes a multiple declaration of 'c10::Device'
19:01:37 C:\Jenkins\workspace\pytorch-builds\pytorch-win-ws2016-cuda9-cudnn7-py3-build\c10/Device.h(24): note: see declaration of 'c10::Device'